image_embeddings/create-image-embeddings.py

```diff
@@ -44,15 +49,14 @@
 parser.add_argument('--delete_existing', dest='delete_existing', required=False, default=True,
                     action=argparse.BooleanOptionalAction,
                     help="Delete existing indices if they are present in the cluster. Default: True")
-parser.add_argument('--ca_certs', dest='ca_certs', required=False, default=CA_CERT,
+parser.add_argument('--ca_certs', dest='ca_certs', required=False, default=False,
```

Comment:
This change is incorrect. The parameter is the path to CA certificates. On the other side, the parameter '--verify_certs' takes a Boolean value to enable/disable SSL/TLS verification.